internal/repository/meal_repository.go

```
@@ -0,0 +1,137 @@
+
+// 메뉴 아이템 일괄 삽입
+func (r *MealRepository) InsertMenuItems(menuItems []models.MenuItem) error {
+    if len(menuItems) == 0 {
+        return nil
+    }
+    
+    tx, err := r.db.Begin()
+    if err != nil {
+        return fmt.Errorf("failed to begin transaction: %w", err)
+    }
+    defer tx.Rollback()
```

golangci-lint (1.64.8)

Error return value of `tx.Rollback` is not checked

(errcheck)